core/services/ardupilot_manager/main.py

```diff
@@ -1,6 +1,34 @@
 #! /usr/bin/env python3
+from typing import Any, Dict, List
+
+import uvicorn
 from ArduPilotManager import ArduPilotManager
+from fastapi import FastAPI
+from mavlink_proxy.Endpoint import Endpoint, EndpointType
+
+app = FastAPI()
+autopilot = ArduPilotManager()
+
+
+@app.get("/available_endpoints", response_model=List[Dict[str, Any]])
+def get_available_endpoints() -> Any:
+    return [endpoint.__dict__ for endpoint in autopilot.get_endpoints()]
+
+
+@app.post("/endpoint", response_model=bool)
+def create_new_endpoint(connection_type: EndpointType, place: str, argument: str = "") -> Any:
+    if autopilot.add_endpoint(Endpoint({"connType": connection_type, "place": place, "argument": argument})):
```

Comment:
We should also follow the api design document and move to snake case:
https://github.com/bluerobotics/companion-docker/wiki/REST-API-design

Reply:
You mean with connType?

Reply:
yes